wallet/wallet.go

```diff
@@ -257,6 +259,36 @@ func (w *Wallet) SetStakeDifficulty(sdi *StakeDifficultyInfo) {
 	w.CurrentStakeDiff = sdi
 }
 
+// GetFeeIncrement is used to get the current feeIncrement for the wallet.
+// Uses non-exported thread safe feeIncrement func.
+func (w *Wallet) GetFeeIncrement() dcrutil.Amount {
+	w.feeIncrementLock.Lock()
+	defer w.feeIncrementLock.Unlock()
+
+	return w.getFeeIncrement()
+}
+
+// feeIncrement is the non-exported mutex safe func to return current
+// w.FeeIncrement
+func (w *Wallet) getFeeIncrement() dcrutil.Amount {
```

Comment:
This is unnecessary and should be removed.  The comment is also incorrect (it is NOT concurrent safe), but that won't matter after it's removed.